08_project/nickolee/cmd/puppy-server/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	puppy "github.com/anz-bank/go-course/08_project/nickolee/pkg/puppy"
+	store "github.com/anz-bank/go-course/08_project/nickolee/pkg/puppy/store"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	// instantiate new puppy store
+	store := store.NewMapStore()
+	// create a puppy and store in store (no pun intended lol)
+	createdPuppy, _ := store.CreatePuppy(&puppy.Puppy{Breed: "The Hound", Colour: "Of Baskerville", Value: 12300.90})
```

Comment:
Get in habbit of not ignoring the errors
In main, this typically looks like
```go
var, err := store.CreatePuppy(...)
if err != nil {
    panic(err)
    // log.Fatal(err) // typical in servers
}
```

Reply:
Thanks for the tip. Will definitely keep that in mind going forward. 

However for the purposes of this lab I couldn't figure out how to hit 100% coverage in my main_test.go so I won't be able to pass CI :sob:

Reply:
Fair enough